python-regression/util/static_vals.py

```diff
@@ -8,11 +8,14 @@
 TEST_NEIGHBORS = [u'tcp://178.128.236.6:14600', u'tcp://167.99.178.3:14600']
 
 TEST_HASH = "NMPXODIWSCTMWRTQ9AI9VROYNFACWCQDXDSJLNC9HKCECBCGQTBUBKVROXZLQWAZRQUGIJTLTMAMSH999"
+TEST_HASH_RES = [True]
 CONFIRMATION_REFERENCE_HASH = "ZSPRRUJRXHBSRFGCCPVHNXWJJKXRYZSAU9ZEGWFD9LPTWOJZARRLOEQYYWIKOPSXIBFD9ADNIVAHKG999"
 NULL_HASH = "999999999999999999999999999999999999999999999999999999999999999999999999999999999"
 
-
 TEST_TIP_LIST = ["SBKWTQWCFTF9DBZHJKQJKU9LXMZD9BMWJIJLZCCZYJFWIBGYYQBJOWWFWIHDEDTIHUB9PMOWZVCPKV999"]
+TEST_HASH_LIST = ["NMPXODIWSCTMWRTQ9AI9VROYNFACWCQDXDSJLNC9HKCECBCGQTBUBKVROXZLQWAZRQUGIJTLTMAMSH999",
+                  "INVALIDWSCTMWRTQ9AI9VROYNFACWCQDXDSJLNC9HKCECBCGQTBUBKVROXZLQWAZRQUGIJTLTMAMSH999"]
```

Comment:
I suppose the second tx is a made up transaction.
Can we also add a real tx that is not included?

I think the scenarios should include:
1. A bundle tail that is approved
2. A bundle tail that is not approved
3. A non-tail that is approved
4. A non-tail that is not apporved
5. a made up tx

Reply:
Dropping scenario 2 and 4 as agreed with @galrogo .